src/Akeneo/Tool/Component/Batch/Item/PausableReaderInterface.php

```diff
@@ -12,4 +12,6 @@
 interface PausableReaderInterface
 {
     public function getState(): array;
+
+    public function rewindToState(int $key): void;
```

Comment:
Rewind from a key is an implementation detail, we should use the state as param
```suggestion
    public function rewindToState(array $state): void;
```